staging/src/k8s.io/client-go/tools/record/events_cache.go

```diff
@@ -37,7 +37,8 @@ const (
 	maxLruCacheEntries = 4096
 
 	// if we see the same event that varies only by message
-	// more than 10 times in a 10 minute period, aggregate the event
+	// more than 10 times, which the max interval of two events are 10 minutes,
```

Comment:
Do you mean "...more than 10 times, and the maximum interval between any two adjacent events is below 10 minutes,..."?

Then this [comment](https://github.com/kubernetes/kubernetes/blob/1945829906546caf867992669a0bfa588edf8be6/staging/src/k8s.io/client-go/tools/record/events_cache.go#L430-L434) is also inaccurate. It mentions the 10 minute "rolling interval".

Reply:
Yes, that's what I mean.
I also fixed the another comment you mentioned.